test/types/bulkCreate.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  Model,
+  InferAttributes,
+  CreationOptional,
+  InferCreationAttributes,
+} from '@sequelize/core';
+import { sequelize } from './connection';
+import type { MakeNullishOptional } from '@sequelize/core/types/utils';
+
+class TestModel extends Model<
+  InferAttributes<TestModel>,
+  InferCreationAttributes<TestModel>
+> {
+  declare id: CreationOptional<number>;
+  declare testString: CreationOptional<string | null>;
+  declare testEnum: CreationOptional<'d' | 'e' | 'f' | null>;
```

Comment:
somewhat recent change: nullable fields are always optional

```suggestion
  declare testString: string | null;
  declare testEnum: 'd' | 'e' | 'f' | null;
```